common.py

```diff
@@ -58,8 +54,7 @@
         'swift-corelibs-xctest': 'swift-4.2-branch',
     },
     'swift-5.0-branch': {
-        'llvm': 'swift-5.0-branch',
-        'clang': 'swift-5.0-branch',
+        'llvm-project': 'swift/swift-5.0-branch',
         'compiler-rt': 'swift-5.0-branch',
```

Comment:
@shahmishal Looks like other configs also remove `compiler-rt`, not sure if that's important or not.

Reply:
I missed `compiler-rt` in this config